dataplex/snippets/src/test/java/dataplex/AspectTypeIT.java

```diff
@@ -0,0 +1,120 @@
+  private static final String LOCATION = "us-central1";
+  private static LocationName locationName;
+  private static AspectTypeName aspectTypeName;
+  private static String expectedAspectType;
+
+  private static final String PROJECT_ID = requireProjectIdEnvVar();
+
+  private static String requireProjectIdEnvVar() {
+    String value = System.getenv("GOOGLE_CLOUD_PROJECT");
+    assertNotNull(
+        "Environment variable GOOGLE_CLOUD_PROJECT is required to perform these tests.", value);
+    return value;
+  }
+
+  @BeforeClass
+  public static void checkRequirements() {
+    requireProjectIdEnvVar();
+  }
+
+  @BeforeClass
+  // Set-up code that will be executed before all tests
+  public static void setUp() throws Exception {
+    String aspectTypeId = "test-aspect-type" + ID;
+    locationName = LocationName.of(PROJECT_ID, LOCATION);
+    aspectTypeName = AspectTypeName.of(PROJECT_ID, LOCATION, aspectTypeId);
+    expectedAspectType =
+        String.format(
+            "projects/%s/locations/%s/aspectTypes/%s", PROJECT_ID, LOCATION, aspectTypeId);
+    // Create Aspect Type resource that will be used in tests for "get", "list" and "update" methods
+    CreateAspectType.createAspectType(locationName, aspectTypeId, new ArrayList<>());
+  }
+
+  @Test
+  public void listAspectTypes_shouldReturnListContainingExpectedAspectType() throws IOException {
+    List<AspectType> aspectTypes = ListAspectTypes.listAspectTypes(locationName);
+    assertThat(aspectTypes.stream().map(AspectType::getName)).contains(expectedAspectType);
+  }
+
+  @Test
+  public void getAspectType_shouldReturnExpectedAspectType() throws IOException {
+    AspectType aspectType = GetAspectType.getAspectType(aspectTypeName);
+    assertThat(aspectType.getName()).isEqualTo(expectedAspectType);
+  }
+
+  @Test
+  public void updateAspectType_shouldReturnUpdatedAspectType() throws Exception {
+    AspectType aspectType = UpdateAspectType.updateAspectType(aspectTypeName, new ArrayList<>());
+    assertThat(aspectType.getName()).isEqualTo(expectedAspectType);
+  }
+
+  @Test
+  public void createAspectType_shouldReturnCreatedAspectType() throws Exception {
+    String aspectTypeIdToCreate = "test-aspect-type" + UUID.randomUUID().toString().substring(0, 8);
+    AspectTypeName aspectTypeNameToCreate =
+        AspectTypeName.of(PROJECT_ID, LOCATION, aspectTypeIdToCreate);
+    String expectedAspectTypeToCreate =
+        String.format(
+            "projects/%s/locations/%s/aspectTypes/%s", PROJECT_ID, LOCATION, aspectTypeIdToCreate);
+
+    AspectType aspectType =
+        CreateAspectType.createAspectType(locationName, aspectTypeIdToCreate, new ArrayList<>());
+    // Clean-up created Aspect Type
+    DeleteAspectType.deleteAspectType(aspectTypeNameToCreate);
+
+    assertThat(aspectType.getName()).isEqualTo(expectedAspectTypeToCreate);
+  }
+
+  @Test
+  public void deleteAspectType_shouldExecuteTheCallWithoutException() throws Exception {
```

Comment:
Please, rename test methods to follow [Java method naming convention](https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names).

Reply:
I am sorry if I miss something, but I don't see how naming convention is violated here, citing:

> Underscores may appear in JUnit test method names to separate logical components of the name, with each component written in lowerCamelCase, for example transferMoney_deductsFromSource. There is no One Correct Way to name test methods.

Could you please be more explicit? I imagine you might refer to `_`, but that is only my guess and it seems to me to be acceptable in the style guide.